v63004/u_inref/gtm7483.csh

```diff
@@ -0,0 +1,38 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+# disable random 4-byte collation header in DT leaf block since this test output is sensitive to DT leaf block layout
+setenv gtm_dirtree_collhdr_always 1
+
+setenv gtm_test_db_format "NO_CHANGE"
```

Comment:
Add comment "# to avoid non-deterministic DBBTUWRNG errors in the mupip integ output"